build.gradle

```diff
@@ -5,9 +5,9 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0-alpha4'
+        classpath 'com.android.tools.build:gradle:2.1.0'
```

Comment:
Sorry, but if someone is not using Studio Android 2.2 preview 4, he can not use this version of the gradle, by the way, the latest version of the AS is 2.2 preview 5 with `2.2.0-alpha5` gradle and cannot use alpha4
![image](https://cloud.githubusercontent.com/assets/5214214/16765377/3e21fe7e-4866-11e6-87b1-6b6eee9036a6.png)